core/src/mindustry/world/blocks/defense/OverdriveProjector.java

```diff
@@ -44,21 +44,13 @@ public OverdriveProjector(String name){
     public boolean outputsItems(){
         return false;
     }
-
+    
```

Comment:
?

Reply:
Dunno why that happened or when I did that.